documentation/modules/ROOT/pages/container-autopsy.adoc

```diff
@@ -0,0 +1,274 @@
+curl -sSL http://ftpmirror.gnu.org/hello/hello-2.10.tar.gz -o hello-2.10.tar.gz
+
+buildah copy $container hello-2.10.tar.gz /tmp/hello-2.10.tar.gz
+
+buildah run $container dnf install -y tar gzip gcc make
+...
+# Workingdir is also a "buildah config" command
+buildah config --workingdir /opt/hello-2.10 $container
+...
+
+# Entrypoint, too, is a “buildah config” command
+buildah config --entrypoint /usr/local/bin/hello $container
+
+# Finally saves the running container to an image
+buildah commit --format docker $container hello:latest
+----
+{zwsp}
+
+=== Podman
+
+Podman command format will be very familiar to Docker users but doesnt require a daemon.
+----
+podman pull registry.fedoraproject.org/f29/httpd
+podman images
+podman run -dt -p 8080:8080/tcp registry.fedoraproject.org/f29/httpd
+----
+
+=== Skopeo
+
+With the skopeo command, you can work with container images from registries without using the docker daemon or the docker command.
+----
+skopeo inspect docker://docker.io/fedora
+
+----
+
+In a single line the Skopeo copy command replaces the mulitple docker commands below
+----
+skopeo copy --dest-creds prod_user:prod_pass docker://internal.registry/myimage:latest docker://production.registry/myimage:v1.0
+----
+
+----
+docker login ...
+docker pull internal.registry/myimage:latest
+docker tag internal.registry/myimage:latest production.registry/myimage:v1.0
+docker push production.registry/myimage:v1.0
+----
+
+{zwsp}
+
+
+[#container_registries]
+== 7. Container Registries
+When working with container images, you need somewhere to save and access them as they are created and that’s where a container registry comes in.
+The registry essentially acts as a place to store container images and share them out via a process of uploading to (pushing) and downloading from (pulling).
+
+There are two types of container registry: public and private.
+
+=== Public registries are:
+
+<.> great for individuals or small teams
+<.> get up and running quickly as possible. Basic in their abilities.
+<.> easy to use
+
+=== Private registries have:
+<.> Security and privacy
+<.> Enterprise container image storage
+<.> Host remotely or on-premises
+
+=== Enterprises need to look for:
+
+<.> Multiple authentication systems
+<.> Role-based access control management
+<.> Vulnerability scanning capabilities
+<.> Auditable logs
+<.> Optimized for automation
+
+Red Hat Quay is a private container image registry that enables you to build, distribute, and deploy containers with the storage you need to scale quickly.
+It analyzes your images for security vulnerabilities using Clair, identifying potential issues and addressing them before they become security risks.
+The Red Hat Container catalog runs on Quay and provides a view of the vulnerabilities found by Clair.
+link:https://catalog.redhat.com/software/containers/explore[https://catalog.redhat.com/software/containers/explore, window=blank]
+
+[#container_exercise]
+== Exercise
```

Comment:
@bfarr-rh As discussed can you remove the exercise bit and instead adapt that for the presentation?

Reply:
done